src/libs/ValidationUtils.js

```diff
@@ -41,7 +40,7 @@ function isRequiredFulfilled(value) {
  * @returns {Boolean} true if valid
  */
 function isValidDate(date) {
-    return moment(date).isValid();
+    return moment(date, 'YYYY-MM-DD', true).isValid();
```

Comment:
Because without the parameters `'YYYY-MM-DD', true`, it becomes more flexible in accepting dates, even to the point of accepting inputs like "223". I think parsing dates so loosely makes it easier for the user to input incorrect data and miss the validation. I thought that adding the strictness was ok because we are telling the user the specific format we are expecting (YYYY-MM-DD).

Reply:
Yeah I guess I can see that logic. Just not sure if this is something we need to solve for yet + also we are adding a date picker soon IIRC.

Reply:
Makes sense, I can remove it.